manifests/init.pp

```diff
@@ -189,6 +192,7 @@
           $default_pid_file        = '/var/run/syslogd.pid'
           $sysconfig_erb           = 'sysconfig.rhel7.erb'
           $default_syslogd_options = '-c 4'
+          $mod_imjournal           = true
```

Comment:
Could you move to this be a parameter with the value 'USE_DEFAULTS' and leave a setting like this for each with `$default_module_imjournal = <true|false>`

Then we can do something like

``` puppet
if $mod_imjournal == 'USE_DEFAULTS' {
  $mod_imjournal_real = $default_mod_imjournal
} else {
  $mod_imjournal_real = $mod_imjournal
}
```